digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/dao/RoadwayDAOSpec.scala

```diff
@@ -1100,5 +1100,39 @@ class RoadwayDAOSpec extends FunSuite with Matchers {
 
     }
   }
+  
+  test("Test When fetching road part history for road address browser then return history information for the road part") {
+    runWithRollback {
+      val roadNumber = 76
+      val roadPartNumber = 1
+      val roadwayNumber = Sequences.nextRoadwayNumber
+      val roadwayNumber2 = Sequences.nextRoadwayNumber
+      val rwHistoryRowStartDate = "1992-10-08"
+      val rwHistoryRowEndDate = "2022-11-14"
+      val rwCurrentRowStartDate = "2022-11-15"
+      val afterChangesSituationDate = "2022-11-15"
+      val historyChangesSituationDate = "2022-11-14"
+
+      val roadways = Seq(
+        Roadway(dao.getNextRoadwayId,	roadwayNumber, roadNumber, roadPartNumber, AdministrativeClass.State, Track.Combined, Discontinuity.Discontinuous, 0, 2080, reversed = false, DateTime.parse(rwHistoryRowStartDate), Some(DateTime.parse(rwHistoryRowEndDate)), "test", Some("TEST ROAD 1"), 8, TerminationCode.NoTermination),
+        Roadway(dao.getNextRoadwayId,	roadwayNumber, roadNumber, roadPartNumber, AdministrativeClass.State, Track.Combined, Discontinuity.Continuous, 0, 2080, reversed = false, DateTime.parse(rwCurrentRowStartDate), None, "test", Some("TEST ROAD 1"), 8, TerminationCode.NoTermination),
+        Roadway(dao.getNextRoadwayId,	roadwayNumber2, roadNumber, roadPartNumber, AdministrativeClass.State, Track.Combined, Discontinuity.Discontinuous, 2080, 2200, reversed = false, DateTime.parse(rwCurrentRowStartDate), None, "test", Some("TEST ROAD 1"), 8, TerminationCode.NoTermination)
+      )
+
+      dao.create(roadways)
+
+      // situation date after changes
+      val result = dao.fetchRoadPartsForRoadAddressBrowser(Some(afterChangesSituationDate), None, Some(roadNumber), Some(roadPartNumber), Some(roadPartNumber))
+      result.size should be (1)
```

Comment:
Jään miettimään, voisiko tähän tulla siis myös jotain muuta kuin "1"? 

Jos, niin milloin?
Jos tulee, niin sellaisellekin vielä testcase.

Jos ei, niin tälle joku kommentti tyyliin // should always return the road part as one line

Reply:
Jos se ei toimi niinkuin pitäisi niin se voisi palauttaa vaikka kaksi riviä

Reply:
Tämän testin datalla siitä pitäisi tulla siis yksi rivi